Clean Swift/Models.xctemplate/___FILEBASENAME___Models.swift

```diff
@@ -52,6 +53,10 @@ enum ___VARIABLE_sceneName___Models {
 
     // MARK: - View Models
 
+    enum AnalyticsEvents {
+        case screenView, primaryButton
```

Comment:
```suggestion
        case screenView
```

We shouldn’t add `primaryButton`; based on historical tasks, it was not needed on all primary buttons.

Reply:
OK